docs/getting-started/migrate/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Migrate from the legacy Shopify App CLI
+section: getting-started
+redirect_from: "/migrate/"
+---
+
+As of version 0.9.0, Shopify App CLI is installed and managed as a software package, instead of as a git repository. If you’re using a legacy version, you’ll need to perform a one-time migration to keep using the CLI. Follow these steps to remove the legacy version and reinstall as a package.
+
+## Check whether you’re using the legacy version
+
+Prior to version 0.9.0, Shopify App CLI was installed as a Git repository. You can determine if you’re running a legacy version by running this command:
+
+```console
+$ shopify --version
```

Comment:
@mkevinosullivan Is this correct? Or should it be `shopify version`?

Reply:
`shopify version`, in keeping with the other commands (`shopify help`, `shopify create`, `shopify connect`, etc.)

Reply:
dunzo